src/services/fungibleTokens.js

```diff
@@ -0,0 +1,81 @@
+import BN from 'bn.js';
```

Comment:
1. Lets export the class as the `default` export so that consumers of the class don't need to use a named import when the class export  is the _primary_ and _only_ reason the file exists
2. I'd like for us to start following airbnb's styling convention for file naming for consistency and readability -- that means making the filename here match the casing of the `default` export -- in this case, PascalCase so they match the name of the class itself (easier indexing and searching) (ref: https://github.com/airbnb/javascript#naming--filename-matches-export).

Reply:
Done.